humility-core/src/hubris.rs

```diff
@@ -401,7 +401,13 @@ pub struct HubrisFlashMap {
 }
 
 impl HubrisFlashMap {
+    /// # Errors
+    ///
+    /// This will fail if the `HubrisArchive` is fake, i.e. contains zero bytes.
```

Comment:
I could add a statement that this list of errors is non-exhaustive... I just didn't want this to be completely undocumented and send someone else rooting around for it.

Reply:
I definitely agree that at least writing down this error case is better! It might be nice to at least mention that the archive being empty is not the only reason, and move on.